test/lib/edamam_api_wrapper_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'test_helper'
+
+describe EdamamApiWrapper do
+
+  describe 'find_recipes' do
+
+    it "Receives a list of recipes from Edamam when given a search-term" do
+      VCR.use_cassette("recipes") do
+        response = EdamamApiWrapper.find_recipes("chicken")
+
+        response.must_be_kind_of Array
+        assert response.any?
+      end
+    end
+
+    it "Can receive valid data from Edamam for a recipe and record it in a array of Recipe objects" do
+      VCR.use_cassette("recipes") do
+        response = EdamamApiWrapper.find_recipes("chicken")
+
+        response[0].must_be_kind_of Recipe
```

Comment:
You should loop through the array and verify that all elements are instances of `Recipe`.